Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -241,14 +241,14 @@ protected function getOpenInvoiceData($order): array
         // Discount cost
         if ($discountAmount != 0) {
 
-            $description = __('Total Discount');
-            $itemAmount = $this->adyenHelper->formatAmount($discountAmount, $currency);
+            $description = __('Discount');
+            $itemAmount = '-'.$this->adyenHelper->formatAmount($discountAmount, $currency);
```

Comment:
I think it's not really readable this way.
Maybe like this?
```suggestion
            $itemAmount = $this->adyenHelper->formatAmount(-1 * $discountAmount, $currency);
```

Reply:
I used the first approach